### ISSUE TITLE
Cannot find module 'babel-runtime/helpers/classCallCheck'

### ISSUE THREAD

**Issue by iFedaka** [RallyTools/rally-node#22]
After fresh install.
I got an error that babel-runtime is not installed.

```
module.js:457
    throw err;
    ^

Error: Cannot find module 'babel-runtime/helpers/classCallCheck'
    at Function.Module._resolveFilename (module.js:455:15)
    at Function.Module._load (module.js:403:25)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (C:\src\node_modules\rally\dist\restapi.js:7:24)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
```

Works in case babel-runtime installed either globally or locally.

**Comment by krmorse** [comment]
Thanks for reporting this.  My guess is we had it installed globally and just missed adding it to package.json.  Expect a fix shortly!

**Comment by pbomb** [comment]
@iFedaka let me know if you still see issues with version 2.1.1, which I just published.

**Comment by iFedaka** [comment]
Nope. Issue is still reproducible.
I'd add it as regular dependency, not as devDependency.

Reproducible in as easy script as:
var rally = require('rally')

On fresh machine with installed node js.

**Comment by pbomb** [comment]
@iFedaka I just published 2.1.2 and tested it locally. Sorry for the mixup.

**Comment by iFedaka** [comment]
Thank you so much.
Now it works, perfectly.
